src/cms/static/js/revisions.js

```diff
@@ -40,7 +40,7 @@ function handle_revision_slider_input(event) {
     // The last updated date of the revision
     const revision_date = u("#revision-" + current_revision).data("date");
     // Update the revision info box
-    revision_info.innerHTML = "Revision: " + current_revision + "<br>" + revision_date;
+    revision_info.textContent = `Revision: ${current_revision}\r\n${revision_date}`;
```

Comment:
Nice, I didn't know this syntax :smile: 
However, it seems that [this is not supported in IE11](https://kangax.github.io/compat-table/es6/#test-template_literals), so maybe we should use normal string concatenation here?

Reply:
Sad :( But okay, I'll revert it :D

Reply:
Oh, but wait: The revisions.js in going through browserify anyway (because of the htmldiff), so I think this is converted to something IE11 understands... :laughing:

Reply:
Indeed, I just checked: after executing
```
npx browserify src/cms/static/js/revisions.js -o src/cms/static/js/revisions_browserified.js -t [ babelify --presets [ @babel/preset-env ] ]
```
the file `revisions_browserified.js` contains the line:
```
revision_info.textContent = "Revision: ".concat(current_revision, "\r\n").concat(revision_date);
```

Reply:
So, we can use this syntax in this file, but not in the rest of the project, right?